docs/guides/main.rst

```diff
@@ -99,19 +103,18 @@ get it right:
    box. If you do that, you can skip the next step.
 
 2. If you haven't done so already, set your ``PATH`` environment variable to
-   include Python and its scripts. To do so, open the "Settings" application, 
-   then access the "System" screen, then access the "About" tab, and then hit 
-   "Advanced system settings" located on the right side of the screen. This 
-   should open the "System Properties" screen, then select the "Advanced" tab, 
-   then hit the "Environmental Variables..." button, and then look for the PATH 
-   variable in the table. Add the following to the end of the variable's value: 
-   ``;C:\Python37;C:\Python37\Scripts``. You may need to adjust these paths to 
-   point to your Python installation.
+   include Python and its scripts. To do so, press Win + R to open the Run dialog
+   box. Next, type ``sysdm.cpl`` to open "System Properties". After, under the 
+   “Advanced” tab, click the “Environment Variables” button, and look for the
+   ``PATH`` variable in the table. Add the following to the end of the variable's
+   value: ``;C:\Python37;C:\Python37\Scripts``. You may need to adjust these paths
+   to point to your Python installation.
```

Comment:
I like this change. Not much of an Windows expert but I think using sysdm.cpl tool directly is the more sustainable method! Nice!